components/generic/DownloadButton.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <b-button
+    :href="downloadUrl"
+    :disabled="downloadDisabled"
+    variant="light text-decoration-none"
+    data-qa="download button"
+    size="lg"
+    class="download-button mr-lg-3 d-inline-flex align-items-center"
+    :target="!useProxy ? '_blank' : '_self'"
+  >
+    <span class="ic-download d-inline-flex pr-1" />
+    {{ $t('actions.download') }}
+  </b-button>
+</template>
+
+<script>
+  export default {
+    props: {
+      downloadUrl: {
+        type: String,
+        default: ''
+      },
+      downloadDisabled: {
+        type: Boolean,
+        default: false
+      },
+      useProxy: {
```

Comment:
With the download button in its own component and having been supplied the constructed URL -- proxied or otherwise -- it need not be aware of whether there is a proxy.

Therefore rename this to `target`, type `String`, and default "_blank".

Reply:
yes good point